File: lsd-interceptors/src/test/java/io/lsdconsulting/interceptors/http/naming/AppNameTest.java
package io.lsdconsulting.interceptors.http.naming;

import io.lsdconsulting.interceptors.common.AppName;
import org.junit.jupiter.api.Test;

import static org.assertj.core.api.AssertionsForClassTypes.assertThat;

class AppNameTest {

    @Test
    void removesSpecialCharacters() {
        var alwaysAppName = new AlwaysAppName(AppName.Factory.create("App/Service (Dev)"));

        assertThat(alwaysAppName.mapForPath("/")).isEqualTo("AppServiceDev");
    }
}

File: lsd-interceptors/src/test/kotlin/io/lsdconsulting/interceptors/http/naming/AppNameTest.kt
package io.lsdconsulting.interceptors.http.naming

import io.lsdconsulting.interceptors.common.AppName.Factory.create
import org.assertj.core.api.AssertionsForClassTypes.assertThat
import org.junit.jupiter.api.Test

internal class AppNameTest {

    @Test
    fun removesSpecialCharacters() {
        val alwaysAppName = AlwaysAppName(create("App/Service (Dev)"))
        assertThat(alwaysAppName.mapForPath("/")).isEqualTo("AppServiceDev")
    }
}